apps/web/pages/signup.tsx

```diff
@@ -246,6 +271,8 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
         email: verificationToken.identifier,
         username,
       },
+      orgSlug:
+        (orgInfo?.organization && (orgInfo?.organization.slug || userOrgMetadata?.requestedSlug)) ?? null,
```

Comment:
```suggestion
      orgSlug:
        orgInfo?.organization?.slug || userOrgMetadata?.requestedSlug ?? null,
```
We can simplify first sentence